keyboards/uranuma/readme.md

```diff
@@ -0,0 +1,15 @@
+# uranuma
+
+It is a keyboard to input using fingers and palms.
+
+A short description of the keyboard/project
+
+Keyboard Maintainer: yohewi(yohewi@gmail.com)
+Hardware Supported: UraNuma, PCB
+Hardware Availability: Case Data
```

Comment:
You should include a link (purchase or GitHub repository) here.

Reply:
it has changed.